Test.C

```diff
@@ -11,108 +11,214 @@
     EncryptionInventory *next; // the pointer
 };
 
-struct Player {
-    char name[NAME_LENGTH]; // the player's name
+struct Player
+{
+    char name[NAME_LENGTH];                      // the player's name
     int equipmentInventory[NUM_EQUIPMENT] = {0}; // the current equipment the user owns
-    EncryptionInventory encryptionInventory; // the current encryption methods the user owns
-    int currentCredits; // the current number of credits the user has available
+    EncryptionInventory encryptionInventory;     // the current encryption methods the user owns
+    int currentCredits;                          // the current number of credits the user has available
 };
 
-struct Round {
+struct Round
+{
     int attackTier;
     Attack attack;
 };
 
 //Function prototypes
 void displayBriefingMessage(Player *playerPtr);
 
-int main(){
+void equipmentStore()
+{
+    int equipSelect;
+    int amountSelect;
+    int transactionTotal;
+    char yesNo;
+
+    printf("|-------------------------------- Equipment Store --------------------------------|\n");
+    printf("%s %s %d %d\n", equipmentStock[0][0] equipmentStock[0][1] equiptmentStock[0][2] equiptmentStock[0][3]);
+    printf("%s %s %d %d\n", equipmentStock[1][0] equipmentStock[1][1] equiptmentStock[1][2] equiptmentStock[1][3]);
+    printf("%s %s %d %d\n", equipmentStock[2][0] equipmentStock[2][1] equiptmentStock[2][2] equiptmentStock[2][3]);
+    printf("%s %s %d %d\n", equipmentStock[3][0] equipmentStock[3][1] equiptmentStock[3][2] equiptmentStock[3][3]);
+    printf("%s %s %d %d\n", equipmentStock[4][0] equipmentStock[4][1] equiptmentStock[4][2] equiptmentStock[4][3]);
+    printf("%s %s %d %d\n", equipmentStock[5][0] equipmentStock[5][1] equiptmentStock[5][2] equiptmentStock[5][3]);
+    printf("%s %s %d %d\n", equipmentStock[6][0] equipmentStock[6][1] equiptmentStock[6][2] equiptmentStock[6][3]);
+    printf("%s %s %d %d\n", equipmentStock[7][0] equipmentStock[7][1] equiptmentStock[7][2] equiptmentStock[7][3]);
+    printf("%s %s %d %d\n", equipmentStock[8][0] equipmentStock[8][1] equiptmentStock[8][2] equiptmentStock[8][3]);
+    printf("%s %s %d %d\n", equipmentStock[9][0] equipmentStock[9][1] equiptmentStock[9][2] equiptmentStock[9][3]);
+    printf("\nPlease Select the number of the equipment you would like to buy"); //1-10 input
+    scanf("\n%d", &equipSelect);
+
+    If(equipSelect > 0 && equipSelect < 11) //if in range
+    {
+        printf("You have selected: ");
+        printf("%s", equipmentStock[equipmentselect - 1][0]);
+        printf("\n How many would you like to buy?");
+        scanf("\n%d", &amountSelect);
+        int transactionTotal = equipmentStock[equipmentselect - 1][2] * amountSelect;
+        printf("\nThis will cost: %d ", transactionTotal);
+        if (player.currentCredits != transactionTotal) //credit check
+        {
+            printf("\nInsufficient funds");
+            return; //lazy return lol
+        }
+        printf("\nAre you sure? Y/N");
```

Comment:
It is standard to have the capital letter when presented an option like this to indicate the default response. As shown in your following if statement, you are not checking every possible response, so your code will default to one of them. I suggest making the default _no_ since this is check to make sure the player is thinking through their decision.
```suggestion
        printf("\nAre you sure? Y/n");
```